### PR TITLE
EMBED-1204 demonstrate extensions using code challenge for OAUTH PKCE …

### PR DESCRIPTION
**Pull request by bryans99**
…flow

An additional OAUTH2 login mechanism has been added that uses PKCE code challenge. Auth0 now demonstrates secret key and code challenge. Note that github does not support code challenge at this time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -263,7 +263,7 @@ const githubSignin = async () => {
 
 ##### Auth0 OAUTH2 setup
 
-The Auth0 OAUTH2 mechanism uses the PKCE flow. Create a [Auth0 account](https://auth0.com). Add the Auth0 client id and base URL to your .env file
+The Auth0 OAUTH2 mechanism uses the PKCE flow with secret key and code challenge. Create a [Auth0 account](https://auth0.com). Add the Auth0 client id and base URL to your .env file
```
Comment (fabio-looker) [inline README.md:266]: Suggested: ...uses the Authorization Code grant type with either a secret key or PKCE (code challenge & code verifier)..

File: README.md [hunk 2]
```diff
@@ -278,7 +278,7 @@ The Auth0 application client secret must also be added to the User attributes in
 - domain whitelist - https://{tenant_id}.auth0.com/login/oauth/token
 - default value - Auth0 client secret
 
-See `Auth.tsx` for authorizing use Auth0 OAUTH2 PKCE flow
+See `Auth.tsx` for authorizing use Auth0 OAUTH2 PKCE flow. The following demonstrates use of the secret key.
```
Comment (fabio-looker) [inline README.md:281]: Suggested: ...for authorizing using Auth0 OAUTH2 Authorization Code grant type.

**Review by fabio-looker — COMMENTED** [review]
One general theme of feedback throughout three places. Unless I've misunderstood something, the name that is shared between the secret key and the code verifier cases is "Authorization Code" whereas "PKCE" (which the PR currently uses to label both cases) only applies to the code verifier case

**Review by fabio-looker — APPROVED** [review]
Much clearer now! Thanks!

**Review by whscullin — APPROVED** [review]
LGTM

**Pull request merged**